runtime/crab/src/pallets/bridge_messages.rs

```diff
@@ -71,8 +72,10 @@ impl Config<WithCrabParachainMessages> for Runtime {
 	type MessageDeliveryAndDispatchPayment =
 		FeeMarketPayment<Self, WithCrabParachainFeeMarket, Ring, RootAccountForPayments>;
 	type MessageDispatch = bm_crab_parachain::FromCrabParachainMessageDispatch;
-	type OnDeliveryConfirmed =
-		(FromDarwiniaIssuing, FeeMarketMessageConfirmedHandler<Self, WithCrabParachainFeeMarket>);
```

Comment:
No need `FromDarwiniaIssuing` anymore? Please check it again. @xiaoch05

Reply:
Yes, FromDarwiniaIssuing should be here https://github.com/darwinia-network/darwinia/pull/909/files#diff-5dfd35aab2476309b3633294333cef016c87f35fa1dc21959e1519b6d07666f4L49